app/app/urls.py

```diff
@@ -189,6 +189,7 @@
     re_path(r'^about/?', retail.views.about, name='about'),
     re_path(r'^mission/?', retail.views.mission, name='mission'),
     re_path(r'^vision/?', retail.views.vision, name='vision'),
+    re_path(r'^not_a_token/?', retail.views.not_a_token, name='not_a_token'),
```

Comment:
lol

Reply:
@thelostone-mc first one was "is_not_a_shitcoin" so "not_a_token" is great hahahaha